dashboards-observability/public/components/trace_analytics/components/traces/__tests__/__snapshots__/span_detail_flyout.test.tsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`<SpanDetailFlyout /> spec renders the empty component 1`] = `
     onClose={[MockFunction]}
     size="s"
   >
-    <OuiWindowEvent
+    <EuiWindowEvent
```

Comment:
we can revert this snap file for now if we haven't made any changes to it.(it is causing failure of checks)